Lib/test/test_pdb.py

```diff
@@ -1295,6 +1296,9 @@ def test_pdb_issue_20766():
 
 
 class PdbTestCase(unittest.TestCase):
+    def setUp(self):
```

Comment:
you need to call setUp on super(). Also, how about create a new test class for tests that need this setUp so that you don't apply it to all the tests?

Reply:
> you need to call setUp on super().

Hm, no. Why?

> Also, how about create a new test class for tests that need this setUp so that you don't apply it to all the tests?

Will do!

Reply:
I guess you're right - you don't need to call setUp in this case. I always do, because my codebase has a hierarchy of test classes and I don't want to worry about which TestCase class is my super().

Reply:
Fixed in a2d255227db3f6c4fc00e89101b67db099ca2c00

Reply:
I'm not sure if this need a news entry. What do you think?

Reply:
I guess you don't need the setUp for PdbTestCase now.